### DISCUSSION TITLE
Discussion area for developers 微信讨论群autogen和autogen studio

### DISCUSSION THREAD

**Discussion by xycjscs** [microsoft/autogen#1709] [General]
It could be helpful to have a place for quick discussions like discord. 
Here we made a Chinese WeChat group for people to discuss autogen and autogen studio. 
建了一个微信讨论群研究autogen和studio。
![IMG_2241](https://github.com/microsoft/autogen/assets/21181784/fe5abd53-a13c-463e-a3d7-9e582a54824a)

**Comment by qingyun-wu** [comment]
Thank you for creating this group!

**Reply by UserName-wang** [reply]
Can someone please help to update the code again? thank you!

**Reply by xulegithub** [reply]
微信群还可以加入吗，我需要讨论

**Comment by xycjscs** [comment]
![IMG_0981](https://github.com/microsoft/autogen/assets/21181784/c6c522ed-363e-4123-a610-0eb7de7f5a48)

**Reply by Josephrp** [reply]
i wish i could join or mirror / translate the convos on some channels to get insight :-)

**Reply by xycjscs** [reply]
welcome

**Comment by xycjscs** [comment]
I make another Feishu and QQ group. The QR codes will be available forever.

![飞书20240410-192450](https://github.com/microsoft/autogen/assets/21181784/a41af906-3382-4767-a4a6-f84dc8be447b)
![qrcode_1712748424298](https://github.com/microsoft/autogen/assets/21181784/2dcf1409-2c83-401c-aa7d-694c0fc36483)

**Reply by uu5208** [reply]
hello 想加入群咨询一下autogen相关的问题

**Comment by MatthewZMD** [comment]
还有新的微信二维码吗
